terraform/app_service.tf

```diff
@@ -58,5 +58,6 @@ resource "azurerm_app_service" "app_service" {
     "SENTRY_PROJECT"                    = var.sentry_project
     "SENTRY_AUTH_TOKEN"                 = var.sentry_auth_token
     "ENABLE_SENTRY"                     = "no"
+    "NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN"  = var.valid_email_domains
```

Comment:
Can this just be SIGN_UP_EMAIL_DOMAINS, please, NEXT_PUBLIC isn't necessary.

Reply:
To access the environment variables from the front end / browser, I had to prefix the env variables with NEXT_PUBLIC. Else it will be undefined.  
Let me know if there is a better way of doing this? TIA